src/server.ts

```diff
@@ -43,6 +44,16 @@ const build = (opts: FastifyServerOptions): FastifyInstance => {
     serve: true,
   });
 
+  fastify.register(metricsPlugin, {
+    endpoint: '/prometheus',
```

Comment:
[Default appears to be](https://www.npmjs.com/package/fastify-metrics) `/metrics` which is a standard that Prometheus will expect - suggest going with default instead:

```suggestion
```